scala/revenj-storage/src/main/scala/net/revenj/storage/S3Repository.scala

```diff
@@ -20,7 +21,7 @@ trait S3Repository {
 		mimeType: Option[String] = None,
 		metadata: Map[String, String] = Map.empty): Future[S3]
 
-	def delete(bucket: String, key: String): Future[Unit]
+	def delete(bucket: String, key: String): Future[DeleteObjectResponse]

```

Comment:
This is a breaking change. Not sure if stuff will break, probably not

Reply:
Because for most cases people will just recompile code and most likely not even using this result type.
So its breaking change only in binary kind of way. Unlikely in source recompile kind of way